### PR TITLE
[IMP] project template: add makepot in .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by sbidoul**
@acsone/acsone-odoo-team

### CODE REVIEW

File: acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob [hunk 1]
```diff
@@ -70,6 +74,7 @@ test:
     - odoo-{{{ odoo.series }}}
   script:
     - install_acsoo
+    - start_ssh_agent  # for pushing translations
```
Comment (benwillig) [inline acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob]: Maybe we could call ```start_ssh_agent``` in the ```makepot``` function?

File: acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob [hunk 2]
```diff
@@ -16,6 +16,10 @@ before_script:
         ssh-add <(echo "$SSH_DEPLOY_KEY")
         echo "${SSH_AGENT_PID}" > ssh-agent.pid
     }
+    function makepot {
+        venv/bin/click-odoo-makepot -c odoo-ci.cfg -d ${DB_NAME} --commit --msgmerge --addons-dir=$(pwd)/odoo/addons
+        git push "git@gitlab.acsone.eu:${CI_PROJECT_NAMESPACE}/${CI_PROJECT_NAME}.git" "HEAD:${CI_COMMIT_REF_NAME}"
```
Comment (ThomasBinsfeld) [inline acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob]: @sbidoul  `${CI_PROJECT_NAMESPACE}/${CI_PROJECT_NAME}` could be written: `${CI_PROJECT_PATH}`
https://docs.gitlab.com/ee/ci/variables/#predefined-variables-environment-variables

File: acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob (hunk 1 above)
Comment (sbidoul) [reply acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob]: The problem I see with that is it hides `start_ssh_agent` a little bit outside of the main code path, and therefore there is the risk to start it twice. We could improve start_ssh_agent to make it reentrant safe, though.

File: acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob (hunk 2 above)
Comment (sbidoul) [reply acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob]: @ThomasBinsfeld done, tested.

**Comment by sbidoul** [comment]
Merged manually

**Pull request closed without merging**